src/geo/ui/infowindow.js

```diff
@@ -185,13 +185,13 @@ cdb.geo.ui.InfowindowModel = Backbone.Model.extend({
     options = options || {};
     var render_fields = [];
     for(var j = 0; j < fields.length; ++j) {
-      var f = fields[j];
-      var value = String(attributes[f.name]);
```

Comment:
@javisantana, we have checked this parameter, but we don't know if there is something behind the shadows, do you have any idea about it?